### ISSUE TITLE
[UX] Review platform updates to accessibility testing reqs

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#51719]
## Tasks

- [x] [Review platform updates to accessibility testing reqs](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/change-history#Changehistory-December1,2022). Review presentation from Brian on these updates. [Conversation in slack](https://dsva.slack.com/archives/C0NGDDXME/p1671476306021549?thread_ts=1671458405.386519&cid=C0NGDDXME). 
- [x] Determine how our process should change or evolve and determine what tickets need to be created. Or, for example, do we always need to create a new ticket for each epic? Outline the changes that we need to make and prepare to discuss with appropriate teams or team members. If changes should be made to our test rail scripts for travel, those changes should be made too. Those tickets are in the Travel Reimbursement - MVP epic. 

## AC

- [x] Team is aware of changes to our process
- [x] Changes to our process have been made and/or communicated to appropriate team members with next action steps

**Comment by kellysmith1008** [comment]
Created [a mural](https://app.mural.co/t/vfscie8528/m/vfscie8528/1673539959850/6377d8f1b2306bab2fa0bc817a44b35302ca315e?sender=u4f51c3ab811a3cc570d34701) outlining the new process. Next step is to determine the answers to the second bullet in the description. Should be completed by Monday

**Comment by kellysmith1008** [comment]
Test rail results (minus voiceover) used for comparison to test artifact
[https://app.zenhub.com/files/133843125/644af8a8-3f78-42e1-957a-b9ba875c4b0f/download](https://app.zenhub.com/files/133843125/644af8a8-3f78-42e1-957a-b9ba875c4b0f/download)

**Comment by kellysmith1008** [comment]
I shared the mural with the team. Lori updated the test rail tests with the color contrast test and updated the DoD on the zoom tests. Moving forward, we'll need to use the test rail tests to include in our AX artifact.
